chapters/inheritance.tex

```diff
@@ -801,15 +802,12 @@ \subsection{Constraining Type}\label{constraining-type}
 and in any subsequent redeclaration. The precedence order is that
 declaration modifiers override constraining type modifiers.
 
-If the \lstinline!constraining-clause! is not present in the original declaration
-(i.e., the non-redeclared declaration):
+If the \lstinline!constraining-clause! is not present in the original declaration (i.e., the non-redeclared declaration):
 \begin{itemize}
 \item
   The type of the declaration is also used as a constraining type.
 \item
-  The modifiers for subsequent redeclarations and constraining type are
-  the modifiers on the component or short-class-definition if that is
-  used in the original declaration, otherwise empty.
+  The modifiers for subsequent redeclarations and constraining type are the modifiers on the component or short class definition if that is used in the original declaration, otherwise empty.
```

Comment:
I also believe this is the grammar construct short-class-definition, since the modifiers are part of that grammar construct.

Reply:
Fixed.  Please resolve if good.